### PR TITLE
[Term Entry] Python Strings: .rjust()

### LINKED ISSUE #3333: [Term Entry] Python Strings .rjust()
### Reason/inspiration (optional)

We would like a new term entry on the `.rjust()` method for Python: [Strings](https://www.codecademy.com/resources/docs/python/strings). The entry should go in a new file under `docs/content/python/concepts/strings/terms/rjust/rjust.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```py `code snippet.
- A “Codebyte Example” section with a ` ```codebyte/python ` snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by noahpgordon**
### Description

Added term for string method **.rjust()** in Python. Closes #3333 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/rjust/rjust.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+Title: '.rjust()'
+Description: 'Adds padding to the left of the parameter string.'
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rjust/rjust.md:3]:
```suggestion
Description: 'Adds padding to the left of the given string.'
```

File: content/python/concepts/strings/terms/rjust/rjust.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The method **`.rjust()`** adds padding of a specified length to the left of a parameter string. The user can specify a character to use as the padding. The number of padding characters added is equal to the length specified in the method call minus the length of the parameter string.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rjust/rjust.md:15]:
```suggestion
The method **`.rjust()`** adds padding of a specified length to the left of a given string. The user can specify a character to use as the padding. The number of padding characters added is equal to the length specified in the method call minus the length of the string provided.
```

File: content/python/concepts/strings/terms/rjust/rjust.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Syntax
+
+```pseudo
+parameter_string.rjust(length, padding_character)
+```
+
+### Required Parameters
+
+ - `parameter_string`: This is the string on which padding will be added.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rjust/rjust.md:25]:
```suggestion
 - `parameter_string`: The string to be modified with padding.
```

File: content/python/concepts/strings/terms/rjust/rjust.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+ - `length`: An integer used to determine how many characters of padding will be added. The number of padding characters will be equal to `length` minus the length of `string_parameter`.
+
+### Optional Parameters
+
+ - `padding_character`: The character used as the padding. Must be a single character. Default value is a space (`" "`).
+
+## Example
+
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rjust/rjust.md:33]: There should be a brief introductory sentence after the example header

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@noahpgordon I've completed my initial review. Please update the entry and tag me when you've finished making changes. Thanks

File: content/python/concepts/strings/terms/rjust/rjust.md (hunk 4 above)
Comment (noahpgordon) [reply content/python/concepts/strings/terms/rjust/rjust.md:33]: Added.

**Comment by noahpgordon** [comment]
@caupolicandiaz I've made the edits.

**Review by caupolicandiaz — APPROVED** [review]
@noahpgordon This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@noahpgordon Looks good.

**Pull request merged**